components/WorkoutList/WorkoutList.js

```diff
@@ -18,21 +23,102 @@ export default function WorkoutList() {
     }
   }
 
-  return (<>
+
+  return (<CardContainer>
     {data.map(workout => (
-      <div key={workout._id}>
-        <h2>{workout.name}</h2>
-        <ul>
-          {workout.exercises.map(({ exercise, sets, reps }) => (
-            <li key={exercise._id}>
-              <p>{exercise.name}</p>
-              <p>Sets: {sets}</p>
-              <p>Reps: {reps}</p>
-            </li>
+      <Card key={workout._id}>
+        <ListContainer>
+          <h2>{workout.name}</h2>
+          {workout.exercises.map(({ exercise, sets, reps, weight }, index) => (<Fragment key={exercise._id}>
+            <ListItem>
+              <Textwrapper>
+                <Text>{exercise.name}</Text>
+                {weight && <Text style={{ color: 'grey', fontSize: '' }}>{exercise.difficulty}</Text>}
+              </Textwrapper>
+              <Textwrapper>
+                <Text2>{sets} x {reps}</Text2>
+                {weight && <Text2 style={{ color: 'grey', fontSize: '' }}>{weight}kg</Text2>}
+              </Textwrapper>
+            </ListItem>
+            {index < workout.exercises.length - 1 && <Divider />}
+          </Fragment>
           ))}
-        </ul>
-        <button onClick={() => handleDelete(workout._id)}>Delete</button>
-      </div>))}
-  </>
+        </ListContainer>
+        <Buttons>
+
+          <DeleteButton onClick={() => handleDelete(workout._id)}><FontAwesomeIcon icon={faTrash} /></DeleteButton>
+        </Buttons>
+      </Card>))}
+  </CardContainer>
   )
-}
\ No newline at end of file
+}
+
+const Textwrapper = styled.div`
+margin: 0;
+display: flex;
+gap: 5px;
+flex-direction: column;`
+
+const Text = styled.span`
+margin: 0;
+font-weight: 200;
+text-align: left;`
+
+const Text2 = styled.span`
+margin: 0;
+font-weight: 200;
+text-align: right;`
+
```

Comment:
Those two components do exactly the same, why not just have a single styled component?